lib/teams.rb

```diff
@@ -0,0 +1,32 @@
+class Team
+  attr_reader :team_id,
+              :franchise_id,
+              :team_name,
+              :abbreviation,
+              :stadium,
+              :link
+
+
+  def initialize(data)
+    @team_id = data[:team_id]
+    @franchise_id = data[:franchiseid]
+    @team_name = data[:teamname]
+    @abbreviation = data[:abbreviation]
+    @stadium = data[:stadium]
+    @link = data[:link]
+  end
+end
+
```

Comment:
Object template matches exactly the headers in CSV file. I don't see a better way to map this out.